packages/core/admin/admin/src/tests/StrapiApp.test.js

```diff
@@ -240,4 +239,83 @@ describe('ADMIN | StrapiApp', () => {
       expect(app.menu[0]).toEqual(link);
     });
   });
+
+  describe('createCustomConfigurations', () => {
+    it('should add a locale', () => {
+      const adminConfig = {
+        config: { locales: ['fr'] },
+      };
+      const app = StrapiApp({ middlewares, reducers, library, adminConfig });
+
+      app.createCustomConfigurations();
+
+      expect(app.configurations.locales).toEqual(['en', 'fr']);
+    });
+
+    it('should override the authLogo', () => {
+      const adminConfig = {
+        config: { auth: { logo: 'fr' } },
+      };
+      const app = StrapiApp({ middlewares, reducers, library, adminConfig });
+
+      app.createCustomConfigurations();
+
+      expect(app.configurations.authLogo).toEqual('fr');
```

Comment:
When you're comparing primitive values with jest, you can do "toBe" instead of toEqual (that does a deep equality)